lab-james/lib/fs-read.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
Generally in your lib files you want to be generic at least with what you are passing in. As of right now this is just a function, that you aren't necessarily handling asynchronously.